angular/projects/spark-angular/src/lib/directives/sprk-button/sprk-button.directive.ts

```diff
@@ -19,12 +19,14 @@ export class SprkButtonDirective implements OnInit, OnChanges, AfterViewInit {
    */
   constructor(public ref: ElementRef, private renderer: Renderer2) {}
 
+  // TODO: Remove spinning functionality from button on next release #3561
   /**
    * Will show a spinner inside the
```

Comment:
We'll want to put the deprecation notice here in the comment so it shows in the prop tables for consumers. Could be useful to mention the new way of doing it or link to the top of the page where we mention the additional engineering or something like that.

Reply:
✅